rskj-core/src/main/java/co/rsk/cli/tools/StartBootstrap.java

```diff
@@ -54,19 +56,22 @@ public StartBootstrap(RskContext ctx) {
 
     public static void main(String[] args) {
         setUpThread(Thread.currentThread());
+        RskCli rskCli = new RskCli();
```

Comment:
If you address the comment above you could simplify this piece by following the same pattern.